src/test/rustdoc-ui/macro-docs.stderr

```diff
@@ -0,0 +1,20 @@
+warning: unresolved link to `long_cat`
+  --> $DIR/macro-docs.rs:5:9
+   |
+LL |         /// A
+   |         ^^^^^
```

Comment:
This seems like there’s another bug hidden in here. It shouldn’t be pointing at this line, should it?